File: backend/FlatSwap/Models/Transcription.cs
using static System.Collections.Specialized.BitVector32;

namespace FlatSwap.Models
{
    public class Transcription
    {
        public int Id { get; set; }

        public required string TranscribedText { get; set; }

        public required string Frequency { get; set; }

        public DateTime Timestamp { get; set; } = DateTime.UtcNow;

        public int StationId { get; set; }

        public required Station Station { get; set; }
    }
}

File: listener-transcriber/Models/transcription.py
from sqlalchemy import Column, Integer, String, Float, DateTime, ForeignKey
from sqlalchemy.ext.declarative import declarative_base

from Models.stations import Stations

Base = declarative_base()

class Transcriptions(Base):
    __tablename__ = 'Transcriptions'
    Id = Column(Integer, primary_key=True)
    StationId = Column('StationId',Integer, ForeignKey(Stations.Id), nullable=False)
    TranscribedText = Column(String, nullable=False)
    Frequency = Column(String, nullable=False)
    Timestamp = Column(DateTime, nullable=False)